internal/resource/config/administrativeaccount/administrative_account_resource.go

```diff
@@ -182,10 +196,24 @@ func (r *administrativeAccountsResource) Configure(_ context.Context, req resour
 
 }
 
-func readAdministrativeAccountResponse(ctx context.Context, r *client.AdministrativeAccount, state *administrativeAccountResourceModel, expectedValues *administrativeAccountResourceModel, passwordPlan basetypes.StringValue) {
+func readAdministrativeAccountResponse(ctx context.Context, r *client.AdministrativeAccount, state *administrativeAccountResourceModel, plan *administrativeAccountResourceModel) {
 	state.Id = types.StringValue(r.Username)
 	state.Username = types.StringValue(r.Username)
-	state.Password = types.StringValue(passwordPlan.ValueString())
+
+	// state.Password
+	if internaltypes.IsDefined(plan.Password) {
```

Comment:
This if check might not be needed if we change password to required, since it will always be there in the plan.